common/stream-model.cpp

```diff
@@ -1510,7 +1536,7 @@ namespace rs2
     void stream_model::show_frame(const rect& stream_rect, const mouse_info& g, std::string& error_message)
     {
         auto zoom_val = 1.f;
-        if (stream_rect.contains(g.cursor))
+        if (stream_rect.contains(g.cursor) && !show_metadata)
```

Comment:
Add a comment please

Reply:
Done

Reply:
Where?

Reply:
Now I see, you placed a comment 500 lines above.
Please add a comment here.
And the main reason I wanted a comment is to explain why we only calculate the mouse scroll for zoom when we don't have metadata.
Something like:

```
`We allow scrolling for zoom in/out when no metadata is on.
If we have metadata on it will scroll the metadata
```

Reply:
Oh, my bad. Those lines look a little similar...

Reply:
Still missing comment.
`Allow mouse scrolling for zoom when not displaying scrollable metadata`

Reply:
The comment was added.